_posts/2018-01-25-review-things-3.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Review: Things 3 for Mac and iOS"
+subtitle: A task manager you will actually want to use
+custom_type: post
+review:
+  rating: 4.5
+  type: app
+  app:
+    developer: Cultured Code
+  price: 9.99-$49.99
+  vendor: Mac App Store
+  url: https://itunes.apple.com/us/app/things-3/id904280696?mt=12
+---
+
+{% figure extendout /uploads/2018/01/things-review-01.jpg "Things 3 on iPad." %}
+
+I've never been a fan of todo apps. I just don't use them. People everywhere seem to swear by these things. There's much talk about the whole GTD mentality, and what app was the best for getting things done.
```

Comment:
Tense disagreement:

"There's much talk" (There _is_)

"and what app was the best" (_was_)